### PR TITLE
[Marketplace] LPS-189777 Create solution template for Clarity solution trial 

### PR DESCRIPTION
**Pull request by ytuik**

### CODE REVIEW

**Comment by ytuik** [comment]
ci:test:sf

**Comment by ytuik** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		</button>
+
+		<button aria-label="Free 30 day trial" class="banner__free-trial-button btn btn-primary px-3 py-3" onclick="Liferay.fire('start-trial');">
+			Free 30 day trial
+		</button>
+	</div>
+</div>
+
+<script>
+	var publisherEmail = "${publisherEmail}";
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.html:48]: What do you need this variable?

**Review by silasbsi — APPROVED** [review]
Good job

**Review by welitonpe — APPROVED** [review]
Good Job :fire:

**Comment by silasbsi** [comment]
ci:forward

**Comment by welitonpe** [comment]
cd:forward

**Comment by ytuik** [comment]
ci:forward

**Pull request closed without merging**